components/cam/src/physics/crm/crm_module.F90

```diff
@@ -30,7 +30,13 @@ module crm_module
   use crm_input_module,       only: crm_input_type
   use crm_output_module,      only: crm_output_type
   use crm_ecpp_output_module, only: crm_ecpp_output_type
-
+  ! The two use commands below are used to in the subroutin crm. 
+  ! The module could not compile them when the two use commands below are put in the original place.  
+  !Putting them here fixed the problem. The reason is not clear, though. 
+  use phys_grid             , only: get_rlon_p, get_rlat_p, get_gcol_p  !, get_gcol_all_p
+#ifdef ECPP  
+  use module_ecpp_crm_driver, only: ecpp_crm_stat, ecpp_crm_init, ecpp_crm_cleanup
+#endif
```

Comment:
This is pretty weird, but I'm not sure we need the a comment with a long explanation. But at the very least jus make sure the indentation is consistent. It drives me crazy to see  inconsistent indentation everywhere.

Reply:
Also, you might as well remove "get_gcol_all_p" since it's not being used.

Reply:
ok. I have removed the comment, which makes it more clean